frontend/src/components/Agreements/AgreementDetailsCards/AgreementTotalCard/AgreementTotalCard.jsx

```diff
@@ -28,7 +28,7 @@ const AgreementTotalCard = ({ total, subtotal, fees, procurementShop }) => {
                     value={subtotal}
                     displayType={"text"}
                     thousandSeparator={true}
-                    decimalScale={2}
+                    decimalScale={subtotal === 0 ? 0 : 2}
```

Comment:
I see this being used quite a bit, it's fairly simple, but curious if we want to wrap it as a formatting constant or something 🤔

Reply:
good callout, probably a good candidate for our constants file

Reply:
maybe something like: 

```javascript
function getDecimalScale(value) {
  return value === 0 ? 0 : 2;
}

// Usage:
<decimalScale={getDecimalScale(data.subtotal)} />
```